### DISCUSSION TITLE
Changing the Icon size

### DISCUSSION THREAD

**Discussion by NeuroticCoder91** [Shopify/polaris-react#8830] [Q&A] [answered]
Hi all,

I want to use icons to represent different content types such as Text, Image, Video. They will be in a card where users can click on them to select the type of content they want. When I use it through the <Icon /> it appears that the size is locked on the viewBox of 0 0 20 20. I couldn't' change this for the life of me. However, when I simply use the icon's component such as <TextMajor /> I am able to control the size, but lose out on the native Props of Icon.  What is the best way to use these icons in this sense?
![Icon source={TextMajor} ](https://user-images.githubusercontent.com/98236590/229378036-7852cde2-dbd8-4eeb-b43e-413fe0d0e8dc.jpg)
Thanks!

**Comment by heyjoethomas** [comment, marked as answer]
Hello. Icons are locked in their size to ensure consistent usage across the Shopify Admin. We currently don't have any way of manipulating the size of an icon.
